lib/src/components/NxTree/NxTree.scss

```diff
@@ -142,10 +142,14 @@
     margin-left: $item-label-left-margin;
     margin-right: var(--nx-spacing-base);
 
-    .nx-icon:not(.nx-tree__colored-icon) {
+    .nx-icon:not(.nx-tree__colored-icon, .nx-icon--colorful) {
       color: var(--nx-swatch-grey-05);
     }
 
+    .nx-tree-key-icon {
+      color: purple;
```

Comment:
This is part of the one gallery example, and needs to be moved to a scss file for just that example.  See other examples that include scss, such as `WithClassExample.tsx`.  The classname should also use a `gallery-` prefix rather than an `nx-` prefix.

Reply:
> This is part of the one gallery example, and needs to be moved to a scss file for just that example. See other examples that include scss, such as `WithClassExample.tsx`. The classname should also use a `gallery-` prefix rather than an `nx-` prefix.

Updated here: 1e0619f0c14757945f2a0141c630eeee2972e602

Reply:
> Updated here: https://github.com/sonatype/sonatype-react-shared-components/commit/1e0619f0c14757945f2a0141c630eeee2972e602

Also include the scss as part of the readable example code.  Again, see `WithClassPage.tsx` to see how.